### PR TITLE
Adding a post of the interview with Joshua

### PR DESCRIPTION
**Pull request by TsukiTiger**
This is the interview recording with Joshua.

### CODE REVIEW

File: _posts/2022-08-08-Interview_Joshua.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## From:
+
+San Jose, CA
+
+It is in the south bay area, the silicon valley. It is hot and does not snow. 
+
+## Living in:
+
+East Bay area. Joshua is living in the city called Doublim.
```
Comment (TsukiTiger) [inline _posts/2022-08-08-Interview_Joshua.md:17]: I found the city in the map. It is where the outlet located in.

Comment (joshuablomgren) [reply _posts/2022-08-08-Interview_Joshua.md:17]: Yeah hahaha you are right!

File: _posts/2022-08-08-Interview_Joshua.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Favorate sport:
+
+As a kid, Joshua liked to play tennis and football.
+
+Right now, Joshua would like to watch swimming.
+
+## Someting more...:
+
+Joshua likes to learn languages. He can speak French, Chinese Mandrain, Korean. And right now he is Learning Spanish. What an amazing passion.
```
Comment (joshuablomgren) [inline _posts/2022-08-08-Interview_Joshua.md:57]: Awwww thank you ! :)

**Review by trevoroldham — APPROVED** [review]
I read it over and it seems good

**Pull request merged**